### PR TITLE
chore: include country selection in POI for EU users

### PR DESCRIPTION
**Pull request by suisin-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [X] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+        .document-dropdown {
+            margin: 0.4rem 0 3.2rem;
+        }
+
+        .country-dropdown {
+            min-height: 35.2rem;
+        }
+    }
+
+    &__fieldset-container {
```
Comment (matin-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:297]: use nesting feature of scss here and for the rest of the changes in the file.
&__fieldset {
   &-container {
   ...
   }
   &-input {
   ...
   }
}

File: packages/core/src/App/Containers/Modals/app-modals.jsx [hunk 2]
```diff
@@ -163,6 +166,10 @@ const AppModals = ({
         ComponentToLoad = <NeedRealAccountForCashierModal />;
     }
 
+    if (is_from_eu) {
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/Modals/app-modals.jsx:169]: lets add a TODO for changing this condition once cashier changes are done.

For testing , lets add is_from_eu and identity and document status are none. ( POI , POA not submitted)

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+                &-container {
+                    margin-top: 1.6rem;
+                    max-width: 50.6rem;
+                    display: flex;
+                    flex-direction: column;
+                }
+
+                &-icon {
+                    margin-right: 1.2rem;
+                    margin-top: 0.2rem;
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:26]: lets use shorthand here

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+                    z-index: 5;
+                }
+            }
+        }
+
+        .icon {
+            width: 128px;
+            height: 128px;
+            margin-top: 5.2rem;
+            margin-bottom: 2.6rem;
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:171]: margin: 5.2rem auto 5.2rem;

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+
+    &__sample-container-external {
+        margin-top: 2.8rem;
+    }
+
+    &__image-container {
+        width: fit-content;
+        height: fit-content;
+        padding: 8px;
+        border-radius: 4px;
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:334]: border-radius: $BORDER_RADIUS;

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+        background-color: $color-grey-2;
+    }
+
+    &__image {
+        max-width: 24.5rem;
+        border-radius: 4px;
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:340]: border-radius: $BORDER_RADIUS;

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+        object-fit: contain;
+
+        @include mobile {
+            max-width: calc(95vw - 16px);
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:344]: rem pls

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 8]
```diff
@@ -0,0 +1,359 @@
...
+    }
+
+    &__sample-container-external {
+        margin-top: 2.8rem;
+    }
+
+    &__image-container {
+        width: fit-content;
+        height: fit-content;
+        padding: 8px;
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:333]: rem pls

File: packages/core/src/App/Containers/VerificationModal/verification-modal.scss [hunk 9]
```diff
@@ -0,0 +1,359 @@
...
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        display: flex;
+        width: 100%;
+        padding: 1.6rem 2.4rem;
+        align-items: center;
+        justify-content: flex-end;
+        z-index: 4;
+        border-radius: 0 0.4rem 0.4rem 0;
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.scss:256]: border-radius: 0 $BORDER_RADIUS $BORDER_RADIUS 0;

File: packages/core/src/App/Containers/VerificationModal/verification-modal.tsx [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+                    toggleModal={() => setIsVerificationModalVisible(false)}
+                    height='700px'
+                    width='996px'
+                    exit_classname='verification-modal--custom-exit'
+                >
+                    <AutoHeightWrapper default_height={200}>
+                        {({ setRef }) => (
+                            <div ref={setRef} className='proof-of-identity'>
+                                <div className='proof-of-identity__main-container'>
+                                    <ProofOfIdentityContainer
```
Comment (amina-deriv) [inline packages/core/src/App/Containers/VerificationModal/verification-modal.tsx:46]: lets move ProofOfIdentityContainer to a separate component like ModalContent. and reuse for Desktop and mobile version

File: packages/stores/types.ts [hunk 11]
```diff
@@ -334,6 +337,7 @@ type TUiStore = {
     setReportsTabIndex: (value: number) => void;
     setIsClosingCreateRealAccountModal: (value: boolean) => void;
     setIsFromSignupAccount: (value: boolean) => void;
+    setIsVerificationModalVisible: (value: boolean) => void;
```
Comment (amina-deriv) [inline packages/stores/types.ts:340]: lets add to mockstore

File: packages/stores/types.ts [hunk 12]
```diff
@@ -322,6 +324,7 @@ type TUiStore = {
     is_dark_mode_on: boolean;
     is_reports_visible: boolean;
     is_language_settings_modal_on: boolean;
+    is_verification_modal_visible: boolean;
```
Comment (amina-deriv) [inline packages/stores/types.ts:327]: lets add to mockstore

**Review by shaheer-deriv — APPROVED** [review]

**Pull request merged**